### DISCUSSION TITLE
How to configure the plugin babel-plugin-date-fns in nextjs 13

### DISCUSSION THREAD

**Discussion by rfw** [vercel/next.js#55467] [Help]
### Summary

How to configure the plugin babel-plugin-date-fns in nextjs 13

**Comment by Nadeera3784** [comment]
Hi @rfw,
I guess we don't need additional  configuration.
We can directly use it. 

`import { differenceInYears, format, formatDistance } from "date-fns";`

According to the documentation there some configuration  you can do. for example

babel
```
{
  "plugins": ["date-fns"],
  "presets": ["@babel/preset-env"]
} 
```
Webpack

```
module: {
  rules: [
    {
      test: /\.m?js$/,
      exclude: /(node_modules|bower_components)/,
      use: {
        loader: 'babel-loader',
        options: {
          presets: ['@babel/preset-env'],
          plugins: ['date-fns']
        }
      }
    }
  ]
}
```

https://github.com/date-fns/babel-plugin-date-fns

**Reply by rfw** [reply]
I configured it this way, but it didn't work

**Reply by Nadeera3784** [reply]
@rfw  Are you getting any errors? also Are you using TypeScript?
